src/compiler.jl

```diff
@@ -3132,11 +3125,14 @@ end
 else
             jctx = ctxToThreadSafe[jctx]
 end
-            cmod, fwdmodenm, _, _ = _thunk(ejob, jctx, #=postopt=#false)
+            cmod, fwdmodenm, _, _ = context!(jctx) do 
```

Comment:
I don't think this should be needed since the calling code should already be in a contet, from fwdfunc, right?